mirobo/vacuum_cli.py

```diff
@@ -294,27 +294,59 @@ def fanspeed(vac: mirobo.Vacuum, speed):
         click.echo("Current fan speed: %s" % vac.fan_speed())
 
 
-@cli.command()
-@click.argument('timer', required=False, default=None)
+@cli.group(invoke_without_command=True)
 @pass_dev
-def timer(vac: mirobo.Vacuum, timer):
-    """Schedule vacuuming, times in GMT."""
-    if timer:
-        raise NotImplementedError()
-        # vac.set_timer(x)
-    else:
-        timers = vac.timer()
-        for idx, timer in enumerate(timers):
-            color = "green" if timer.enabled else "yellow"
-            #  Note ts == ID for changes
-            click.echo(click.style("Timer #%s, id %s (ts: %s)" % (
-                idx, timer.id, timer.ts), bold=True, fg=color))
-            print("  %s" % timer.cron)
-            min, hr, x, y, days = timer.cron.split(' ')
-            # hr is in gmt+8 (chinese time), TODO convert to local
+@click.pass_context
+def timer(ctx, vac: mirobo.Vacuum):
+    """List and modify existing timers."""
+    if ctx.invoked_subcommand is not None:
+        return
+    timers = vac.timer()
+    for idx, timer in enumerate(timers):
+        color = "green" if timer.enabled else "yellow"
+        #  Note ts == ID for changes
+        click.echo(click.style("Timer #%s, id %s (ts: %s)" % (
+            idx, timer.id, timer.ts), bold=True, fg=color))
+        print("  %s" % timer.cron)
+        min, hr, x, y, days = timer.cron.split(' ')
+        # hr is in gmt+8 (chinese time), TODO convert to local
```

Comment:
Since the vacuum supports timezones nowadays, those should be times in active timezone. Removed the conversion and added a print-out for the timezone.